### PR TITLE
Update server URLs in code

### LINKED ISSUE #21: rpecanapi: `ping()` and `get.status()` Functions Return 'Server Not Responding' Error When Connecting to PEcAn Server
### Bug Description

When attempting to use the `rpecanapi` package to connect to the PEcAn server, the functions `ping()` and `get.status()` return "Sorry! Server not responding.", which is the default `error` response in the `tryCatch` statement.

### To Reproduce

1. Load the `rpecanapi` library.
2. Connect to the PEcAn server using the `connect()` function with the appropriate URL, username, and password.
3. Call the `ping()` or `get.status()` function.

## Expected behavior

I expect the `ping()` and `get.status()` functions to return a response from the PEcAn server indicating its status as follows : 

For the `ping(server)` command :

```r
ping(server)

#> $request
#> [1] "ping"

#> $response
#> [1] "pong"
```

For the `get.status(server)` command :

```r
get.status(server)

#> $pecan_details$version
#> [1] "1.7.0"

#> $pecan_details$branch
#> [1] "develop"

#> $pecan_details$gitsha1
#> [1] "unknown"

#> $host_details$hostid
#> [1] 99

#> $host_details$hostname
#> [1] ""

#> $host_details$start
#> [1] 99000000000

#> $host_details$end
#> [1] 99999999999

#> $host_details$sync_url
#> [1] ""

#> $host_details$sync_contact
#> [1] ""

#> $host_details$authentication_required
#> "TRUE"
```

## Screenshots

Screenshot 2024-03-05 at 11 07 34 AM

## Machine (please complete the following information):

 - Server : Localhost
 - OS: MacOS

## Additional context

I'm running the R session on a Linux machine and attempting to connect to a PEcAn server running locally on port 8000. The server is expected to be responsive as per its normal behavior.

### PR DESCRIPTION
**Pull request by Sweetdevil144**
Updated API server URL in code files
Fixes #21 

Priority: Low
Status: Completed
Type: Bug fix

## Description

Replaced all instances of 'http://localhost:8000' with 'http://pecan.localhost:80' wherever required.

## Motivation and Context

This change was required to correct the API server URL in the examples provided in the example file. The incorrect URL was causing connection issues when users tried to follow the examples.

## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [x] When possible

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue) 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by Sweetdevil144** [comment]
Used following bash command to do so : 

```zsh
find . -name '*.Rd' -exec sed -i '' 's|http://localhost:8000|http://pecan.localhost:80|g' {} \; && 
find . -name '*.R' -exec sed -i '' 's|http://localhost:8000|http://pecan.localhost:80|g' {} \; && 
find . -name '*.Rmd' -exec sed -i '' 's|http://localhost:8000|http://pecan.localhost:80|g' {} \;
```

**Review by mdietze — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -52,44 +55,50 @@ ping(server)
 #> $response
 #> [1] "pong"
 ```
+
 #### Get General Information about the Server & PEcAn Version
+
 ```R
-get.status(server)
+# get.status(server)
```
[the remark this answers is not in the document]
Comment (mdietze) [reply README.md]: So I see that you've reverted this particular line, but you still haven't responded to the bigger question about why there are so many changes in this file beyond just the URL change

**Review by Sweetdevil144 — COMMENTED** [review]
I guess the new lines were added by the linter to seperate diffferent markdown blocks

**Comment by Sweetdevil144** [comment]
Any guess why the checks fail?

**Comment by mdietze** [comment]
Nothing you did, just the aging of code, but there need to be some updates to the yml file. If you look at .github/workflows
/pkgdown.yml anywhere you see r-lib/actions the version needs to be updated from @master to @v2. The @master branch was renames @main but @v2 is the current recommended stable branch. See the readme at https://github.com/r-lib/actions

**Comment by Sweetdevil144** [comment]
Sure. I guess it can be fixed by changing the `steps` as below :
```yaml
- uses: actions/checkout@v2

      - uses: r-lib/actions/setup-r@v2 # Updated to use a version tag
      - uses: r-lib/actions/setup-pandoc@v2 # Updated to use a version tag
```

I guess the workflow control is by your side. Or should I make the changes myself?

**Comment by mdietze** [comment]
@Sweetdevil144 you should make the changes yourself as part of this PR to see if that fixes the GH Action checks

**Comment by Sweetdevil144** [comment]
`pkgdown.yml` successfully runs after I opened a PR after making some linter changes : https://github.com/Sweetdevil144/rpecanapi/pull/1

Problem with ci.yml as follow : https://github.com/Sweetdevil144/rpecanapi/actions/runs/8409126911/workflow

Error : `The workflow is not valid. .github/workflows/ci.yml (Line: 59, Col: 9): Unexpected value 'args'`

Maybe I can replace : 
```yml
- name: create account
  uses: docker://pecan/bety:latest
  args: user carya illinois "Carya Demo User" carya@example.com 1 1
```

with 

```yml
- name: create account
  uses: docker://pecan/bety:latest
  with:
    args: user carya illinois "Carya Demo User" carya@example.com 1 1
```

Any suggestions @mdietze ?

**Comment by Sweetdevil144** [comment]
Hi @mdietze . Can you verify the status of this issue?

**Pull request merged**